ocs_ci/ocs/cluster.py

```diff
@@ -1551,6 +1552,34 @@ def is_flexible_scaling_enabled():
     return failure_domain == "host" and flexible_scaling
 
 
+def validate_osd_pdb():
+    """
+    Validate creation of PDBs OSD pods.
+    1. Versions lesser than ocs-operator.v4.6.2 have PDBs for each osds
+    2. Versions greater than or equal to ocs-operator.v4.6.2-233.ci have PDBs collectively for osds like rook-ceph-osd
+    """
+
+    pdb_obj = ocp.OCP(
+        kind="PodDisruptionBudget",
+        selector=constants.OSD_PDB,
+        namespace=defaults.ROOK_CLUSTER_NAMESPACE,
+    )
+    allowed_disruptions = pdb_obj.get().get("status").get("disruptionsAllowed")
```

Comment:
Since you're using pdb_obj.get() 4 times, I think you can hold a variable like pdb_obj_get or similar.
Then, you can just use it like pdb_obj_get.get("spec") for example.